### ISSUE TITLE
Ensure timely conda packaging

### ISSUE THREAD

**Issue by deeplook** [deeplook/svglib#240]
This is mainly for @nicoddemus as he set up the packaging of svglib for conda-forge. ATM one can find 0.9.3 there, see https://anaconda.org/conda-forge/svglib, so I wonder if and what we can do to make sure that new versions are automatically picked up and get published there, as well? Maybe you can describe the process? Thanks!

**Comment by nicoddemus** [comment]
There's a bot which routinely scans PyPI and opens up a PR updating the package on conda-forge, so it should be a matter of a day or two before a new version pops up there.

**Comment by deeplook** [comment]
Yes, but it's seven days now. ;)

**Comment by nicoddemus** [comment]
Yeah, I haven't been keeping up with my packaging duties lately, but I will try to keep up.

**Comment by deeplook** [comment]
No worries! I just want to get a better understanding of what is happening and how, also because I might want to publish other things on conda-forge.
